.travis.yml

```diff
@@ -5,8 +5,7 @@ services:
   - mongodb
   - rabbitmq
   - redis-server
-install:
-  - sudo rabbitmq-plugins enable rabbitmq_mqtt
-  - sudo service rabbitmq-server restart
 env:
   - TERM=dumb SI_FATAL_WHEN_NO_BEANFACTORY=true GRADLE_OPTS='-XX:MaxPermSize=512M -Xmx1024M'
+script:
+  - ./gradlew build --parallel
```

Comment:
M-m-m. Don't see anything similar in Travis logs.
Where to search for proper way to make it working?